salt/modules/zabbix.py

```diff
@@ -1466,7 +1689,7 @@ def hostinterface_get(hostids, **connection_args):
         return ret
 
 
-def hostinterface_create(hostid, ip, dns='', main=1, type=1, useip=1, port=None, **connection_args):
+def hostinterface_create(hostid, ip_, dns='', main=1, if_type=1, useip=1, port=None, **connection_args):
```

Comment:
Should this still be ip_ and if_type?

Reply:
Yes, in this case it is correct since Salt coding style (and pylint) says at least three characters for variable name and "type" is reserved Python keyword.